src/Uno.UWP/ApplicationModel/Core/CoreApplication.netstdref.cs

```diff
@@ -0,0 +1,10 @@
+﻿#nullable enable
+
+namespace Windows.ApplicationModel.Core;
+
+partial class CoreApplication
+{
+	private static void ExitPlatform()
+	{
+	}
```

Comment:
I don't think an empty implementation like that is correct? Also reported issue for Skia so `CoreApplication.Exit` is already implemented.

Regarding `Application.Exit`:

https://github.com/unoplatform/uno/blob/c4288e1062bda783397d2b3145086298883c73d0/src/Uno.UI/UI/Xaml/Application.cs#L231-L245

It's also already implemented in Skia and delegates to `CoreApplication.Exit` and should be working already. (the original issue reported in fact says that it's working)

I think the issue might be that we don't pass specific platforms to `NotImplemented` attribute. I don't recall why the `#if` directives alone are not sufficient though.

Reply:
To my understanding, the fix should be a single line changing the following line:

https://github.com/unoplatform/uno/blob/c4288e1062bda783397d2b3145086298883c73d0/src/Uno.UI/UI/Xaml/Application.cs#L232

to

```csharp
[NotImplemented("__IOS__", "NET461", "__WASM__", "__NETSTD_REFERENCE__")]
```

Reply:
I went ahead and opened https://github.com/unoplatform/uno/pull/10457

Reply:
You are right, the reported warning was most likely caused by that `[NotImplemented]` attribute! I think this one is not needed then.